SpringProject/src/main/java/com/gradproject2019/users/controller/UserController.java

```diff
@@ -19,6 +24,12 @@
     @Autowired
     private UserService userService;
 
+    @GetMapping(path = "/users/search/{searchInput}")
```

Comment:
Search input should be a query parameter, look at 
google.com/search?q=something
That's because the 'something' isn't pointing to a specific resource, it's just filtering the user resource